ga4gh/backend.py

```diff
@@ -384,6 +386,49 @@ def setMaxResponseLength(self, maxResponseLength):
         """
         self._maxResponseLength = maxResponseLength
 
+    # private methods used to implement readsGenerator
```

Comment:
There's only one method here, so a quick docstring would be more appropriate I think. Ditto the other methods below.